src/azure-cli/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -121,22 +125,83 @@ def pg_storage_validator(ns):
+
+def mysql_sku_name_validator(ns):
+    if ns.sku_name:
+        if len(ns.sku_name.split('_')) == 2:
+            server_type, cpu = ns.sku_name.split('_')
+            if server_type != 'Standard' or cpu not in ['B1ms', 'B2s', 'B1s']:
+                raise CLIError('Incorrect value for --sku-name. Follow the convention Standard_{VM name}. Examples: Standard_D4s_v3 Standard_E8s_v3 Standard_B1ms')
+        elif len(ns.sku_name.split('_')) == 3:
+            server_type, cpu, server_v3 = ns.sku_name.split('_')
+            if server_type != 'Standard' or server_v3 != 'v3':
+                raise CLIError('Incorrect value for --sku-name. Follow the convention Standard_{VM name}. Examples: Standard_D4s_v3 Standard_B1ms')
+            if not re.match(r"^(D|E)\d+s$", cpu) or (re.match(r"^(D|E)\d+s$", cpu) and int(cpu[1:-1]) not in [2, 4, 8, 16, 32, 48, 64]):
+                raise CLIError('Incorrect value for --sku-name.  Follow the convention Standard_{VM name}. Examples: Standard_D4s_v3 Standard_E8s_v3 Standard_B1ms. \nAllowed number of vCores: 2, 4, 8, 16, 32, 48, 64')
+
+def pg_version_validator(ns):
+    if ns.version:
+         if ns.version not in ['11', '12']:
+            raise CLIError('Incorrect value for --version. Allowed values : {11, 12}')
+
+def mysql_version_validator(ns):
+    if ns.version:
+         if ns.version not in ['5.6', '5.7', '8.0']:
+            raise CLIError('Incorrect value for --version. Allowed values : {5.6, 5.7, 8.0}')
+
+def maintenance_window_validator(ns):
+    if ns.maintenance_window:
+        parsed_input = ns.maintenance_window.split(':')
+        if len(parsed_input) < 1 or len(parsed_input) > 3:
+            raise CLIError('Incorrect value for --maintenance_window. Enter <Day>:<Hour>:<Minute>. Example: "0:8:30" to schedule on Monday, 8:30 UTC')
+        elif len(parsed_input) >= 1 and (not parsed_input[0].isdigit() or int(parsed_input[0]) < 0 or int(parsed_input[0]) > 6): 
+            raise CLIError('Incorrect value for --maintenance_window. The first number means the scheduled day in a week. Allowed values: {0, 1, 2, 3, 4, 5, 6}')
+        elif len(parsed_input) >= 2 and (not parsed_input[1].isdigit() or int(parsed_input[1]) < 0 or int(parsed_input[1]) > 23): 
+            raise CLIError('Incorrect value for --maintenance_window. The second number means the scheduled hour in the scheduled day. Allowed values: {0, 1, ... 23}')
+        elif len(parsed_input) >= 3 and (not parsed_input[2].isdigit() or int(parsed_input[2]) < 0 or int(parsed_input[2]) > 59): 
+            raise CLIError('Incorrect value for --maintenance_window. The third number means the scheduled minute in the scheduled hour. Allowed values: {0, 1, ... 59}')
+
+def ip_address_validator(ns):
+    if ns.end_ip_address and not _validate_ip(ns.end_ip_address) or ns.start_ip_address and not _validate_ip(ns.start_ip_address):
```

Comment:
Check the behavior. You might need to change this to :
(ns.end_ip_address and not _validate_ip(ns.end_ip_address)) or (ns.start_ip_address and not _validate_ip(ns.start_ip_address)):

Reply:
Fixed! Thanks for the feedback! :)